update readme with realtime doc (#334)
ably/ably-python: README.md

await client.close()
```

### Using the Realtime API
The python realtime API currently only supports authentication with ably API key.
#### Creating a client
```python
from ably import AblyRealtime

async def main():
    client = AblyRealtime('api:key')

>     channel = client.channels.get('channel_name)

Reviewer: Probably remove this line because it's not relevant to creating a client

```

> #### Subscribing to a channel for event

Reviewer: ```suggestion
#### Subscribing to messages on a channel
```

```python

> message_future = asyncio.Future()

Reviewer: I think you should remove the `message_future` bit because that's a little confusing. Just change it so that the listener prints out the `message.data`.

def listener(message):
    message_future.set_result(message)

> channel.subscribe('event', listener)

Reviewer: ```suggestion
# Subscribe to messages with the 'event' name
await channel.subscribe('event', listener)
```

Reviewer: you should also add something to mention that `channel.subscribe` is an coroutine function and will resolve when the channel is attached.

> # Subscribe using only listener

Reviewer: ```suggestion
# Subscribe to all messages on a channel
```

await channel.subscribe(listener)
```

> #### Unsubscribing from a channel for event

Reviewer: ```suggestion
#### Unsubscribing from messages on a channel
```

```python
# unsubscribe the listener from the channel
channel.unsubscribe('event', listener)

# unsubscribe all listeners from the channel
channel.unsubscribe()
```

> #### Attach a channel

Reviewer: ```suggestion
#### Attach to a channel
```

```python
await channel.attach()
```
#### Detach from a channel
```python
await channel.detach()
```

#### Managing a connection
```python
# Establish a realtime connection.
# Explicitly calling connect() is unnecessary unless the autoConnect attribute of the ClientOptions object is false
await client.connect()

# Close a connection
await client.close()

> # Ping a connection

Reviewer: ```suggestion
# Send a ping
```

> await client.connection.ping()

Reviewer: ```suggestion
time_in_ms = await client.connection.ping()
```

await client.close()
```

### Using the Realtime API

> The python realtime API currently only supports authentication with ably API key.

Reviewer: ```suggestion
The python realtime client currently only supports basic authentication.
```

#### Creating a client
```python
from ably import AblyRealtime

async def main():
    client = AblyRealtime('api:key')
```

> #### Connecting to a channel

Reviewer: ```suggestion
#### Get a realtime channel instance
```

```python

> channel = client.channels.get('channel_name)

Reviewer: ```suggestion
channel = client.channels.get('channel_name')
```

channel.unsubscribe()
```

#### Subscribe to connection state change

> ```python

Reviewer: I think we should add `from ably.realtime.connection import ConnectionState` just so that users know where to get the state enum from

## Known Limitations

> Currently, this SDK only supports [Ably REST](https://ably.com/docs/rest) and subscribe/unsubscribe functionality of [Ably Realtime](https://ably.com/docs/realtime) as documented above.

Reviewer: ```suggestion
Currently, this SDK only supports [Ably REST](https://ably.com/docs/rest) and realtime message subscription as documented above.
```
